docs/js/guides/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+
+**Case 1:** When run locally, the SDK reads the destination from the environment variables. The `jwt` is irrelevant.
+The destination contains only the name and URL of the real CF destination. 
+Since the destination has no proxy type specified, the SDK takes the HTTP_PROXY into account, as is the default.
+From there the flow described above takes place.
+
+**Case 2:** When run on CF there is no environment variable present. 
+The `jwt` is used to fetch the full destination from the service.
+The proxy type is `OnPremise` and the connectivity service provides all proxy information.
+
+
+:::note
+The `executeHttpRequest()` function is used by all request builders provided of the SDK as well as by CAP applications connecting to an SAP S/4HANA system.
+:::
+
+## Setup ##
+
+### Prerequisite I: The Business Application Studio ###
+
+You have a Cloud Foundry account and a subscription to the BAS. 
+Start the BAS and connect your BAS workspace to the CF account. 
+This is done via the little CF icon (<img src={useBaseUrl('img/cf-connect-button.jpg')} />) on the left of the BAS.
+The connection enables the proxy connection from the BAS to your CF account.
+
+### Prerequisite II: Cloud Connector and Destination Setup  <a name="CCandDestSetup"></a> ###
+
+You have a working Cloud Connector setup and in your account contains a destination pointing to an On-Premise system you want to connect to.
+Per default, destinations are not usable by a connected BAS. 
+You need to set two properties `WebIDEEnabled` and `HTML5.DynamicDestination` to enable that feature for a specific destination.
+Go to the destination configuration in CF and add the properties:
+
+|Property|Value|
+|---|---|
+|WebIDEEnabled|true|
+|HTML5.DynamicDestination|true|
+  
+### Local Application Setup
+
+You use [launch configurations](https://code.visualstudio.com/docs/editor/debugging#_launchjson-attributes) to run and debug applications locally.
+Either extend your existing `.vscode/launch.json` or create a new one. 
+```JSON
+{"configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug my application",
+      "program": "${workspaceFolder}/dist/main.js",
+      "envFile": "${workspaceFolder}/.env",
+      "preLaunchTask": "npm: build",
+      "outFiles": [
+        "${workspaceFolder}/dist/**/*.js"
+      ]
+    }
+  ]
+}
+```
+In this example we use a simple Nest.js application. 
+The code will look very similar for express or [CAP](https://community.sap.com/topics/cloud-application-programming) applications.
+The property `program` defines the script file executed when you run the configuration. 
+In our case the `main.js` script will start up the Nest server.
+The `preLaunchTask` executes the build before each run which compiles the TypeScript files to JavaScript. 
+The `outFiles` properties defines where the compiled files will be located.
+
+As discussed [in the beginning](#background) we need to set an environment variable.
```

Comment:
Just to make sure: Is this really read automatically? Does the BAS have some magic here? Maybe it would be nice to add, why adding to the env file already does the trick?

Reply:
added an extra sentence and link.